doc/commands/atc.md

```diff
@@ -17,11 +17,13 @@ Executes ATC Checks on the given object and exits with non-zero code,
 if ATC findings of Prio higher then the configured level are found.
 
 ```bash
-sapcli atc run {package,class,program} OBJECT_NAME [-r VARIANT] [-e ERROR_LEVEL] [-m MAX_VERDICITS]
+sapcli atc run {package,class,program} OBJECT_NAME [-r VARIANT] [-e ERROR_LEVEL] [-m MAX_VERDICITS] [-f FORMAT] [-s SEVERITY_MAPPING]
```

Comment:
s/FORMAT/txt|hml|checkstyle/

Reply:
s/-f/--output|-o/

Reply:
I use human instead of txt for aunit: https://github.com/jfilak/sapcli/blob/master/doc/commands/aunit.md